### ISSUE TITLE
No sound with inactive AGC for SSB Demodulator

### ISSUE THREAD

**Issue by Vort** [f4exb/sdrangel#356]
When SSB Demodulator is started, `m_agcActive` is initialized to `false`.
But then, in `SSBDemod::processOneSample`, `m_agc` is accessed, in this line:
```fftfilt::cmplx z = delayedSample * m_agc.getStepValue();```
`m_agc.getStepValue()` returns zero and then `sample` becomes zero too.

**Comment by f4exb** [comment]
Changed the line to: 
```
fftfilt::cmplx z = m_agcActive ? delayedSample * m_agc.getStepValue() : delayedSample
```
Which should fix the main issue however the fixed multiplier had to be changed from 10.0 to 0.1 because the resulting level was far too high. The issue with SSB is that audio level is directly linked to RF level and it can vary by large amounts. I have based these heuristics on your Fox News sample which is representative of a fairly loud signal.  In fact in most cases you should turn on AGC until you know a bit more about your signal and possibly want to turn it off for weak signal work.

I also changed default volume to 1.0 and AGC power threshold (which is a squelch more or less) to -100 dB to be more representative of what you can expect in an average situation.

Now on dev branch.

**Comment by Vort** [comment]
Thanks!
